packages/website/src/pages/ContentPage/DocPage.tsx

```diff
@@ -14,6 +15,9 @@ import { fromHTML, MakeTree } from '../../utils/docHeaders';
 export interface IDocPageProps {
   file: DriveFile;
   renderStackOffset?: number;
+  match: any;
+  location: any;
+  history: any;
```

Comment:
These props seems to be unused. Why do we need this?

BTW if you want DocPage to be re-rendered when router is changed, you can use `useHistory`

Reply:
match is used. The other two are required by the type checker but not used.

Reply:
These are supplied by `withRouter`